app/Views/errors/html/production.php

```diff
@@ -4,21 +4,21 @@
     <meta charset="UTF-8">
     <meta name="robots" content="noindex">
 
-    <title>Whoops!</title>
+    <title><?= lang('Errors.whoops') ?></title>
 
     <style type="text/css">
         <?= preg_replace('#[\r\n\t ]+#', ' ', file_get_contents(__DIR__ . DIRECTORY_SEPARATOR . 'debug.css')) ?>
     </style>
 </head>
 <body>
 
-    <div class="container text-center">
+<div class="container text-center">
```

Comment:
Please revert the indentation.
Or do you have any reason to change the indentation?